rust/Cargo.lock

```diff
@@ -577,15 +577,16 @@ dependencies = [
 
 [[package]]
 name = "bip32"
-version = "0.5.2"
-source = "git+https://github.com/KeystoneHQ/crates.git?rev=9873e8fd56007d792fa60d6e844fdb75d527c858#9873e8fd56007d792fa60d6e844fdb75d527c858"
+version = "0.6.0-pre.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "143f5327f23168716be068f8e1014ba2ea16a6c91e8777bc8927da7b51e1df1f"
```

Comment:
Reviewed https://github.com/KeystoneHQ/crates/compare/main...iqlusioninc:crates:bip32/v0.6.0-pre.1 . Looks to be mostly dependency updates, other changes are trivial. Rust MSRV is bumped to 1.81.0.